packages/manager/src/components/Drawer.tsx

```diff
@@ -92,7 +96,9 @@ export const Drawer = (props: Props) => {
         }
       }}
       anchor="right"
-      classes={{ paper: `${classes.paper} ${wide ? classes.wide : ''}` }}
+      classes={{
+        paper: `${classes.common} ${wide ? classes.wide : classes.default}`,
+      }}
```

Comment:
Drawer behavior looks great now! 🎉

Last bit of feedback: these classnames should be handled with a conditional classnames helper like `cx`. You can get the `cx` function from the useStyles hook like so:  `const { classes, cx } = useStyles();`

```suggestion
      classes={{
        paper: cx(classes.common, {
          [classes.default]: !wide,
          [classes.wide]: wide,
        }),
      }}
```

or 


```suggestion
      classes={{
        paper: cx(classes.common, wide ? classes.wide : classes.default),
      }}
```